test/util.test.js

```diff
@@ -468,4 +470,375 @@ describe('mergeOptions', function () {
+      assert.equal(typeof util.convert(new moment(), 'ISODate'), 'string');
+    });
+
+    it('throws when converting unknown object to ISODate', function () {
+      assert.throws(function () {util.convert({}, 'ISODate');}, Error, null);
+    });
+
+    it('converts to ASPDate from Number', function () {
+      assert(ASPDateRegex.test(util.convert(1198908717056, 'ASPDate')));
+    });
+
+    it('converts to ASPDate from String', function () {
+      assert(ASPDateRegex.test(util.convert('1995-01-01', 'ASPDate')));
+    });
+
+    it('converts to ASPDate from Date', function () {
+      assert(ASPDateRegex.test(util.convert(new Date(), 'ASPDate')));
+    });
+
+    it('converts to ASPDate from ASPDate', function () {
+      assert(ASPDateRegex.test(util.convert('/Date(12344444)/', 'ASPDate')));
+    });
+
+    xit('converts to ASPDate from Moment - skipped, because it fails', function () {
+      assert(ASPDateRegex.test(util.convert(new moment(), 'ASPDate')));
+    });
+
+    it('throws when converting unknown object to ASPDate', function () {
+      assert.throws(function () {util.convert({}, 'ASPDate');}, Error, null);
+    });
+
+    it('throws when converting unknown type', function () {
+      assert.throws(function () {util.convert({}, 'UnknownType');}, Error, null);
+    });
+  });
+  describe('getType', function () {
+    it('of object null is null', function () {
+      assert.equal(util.getType(null), 'null');
+    });
+    it('of object Boolean is Boolean', function () {
+      function Tester () {}
+      Tester.prototype = Object.create(Boolean.prototype);
+      assert.equal(util.getType(new Tester('true')), 'Boolean');
+    });
+    it('of object Number is Number', function () {
+      function Tester () {}
+      Tester.prototype = Object.create(Number.prototype);
+      assert.equal(util.getType(new Tester(1)), 'Number');
+    });
+    it('of object String is String', function () {
+      function Tester () {}
+      Tester.prototype = Object.create(String.prototype);
+      assert.equal(util.getType(new Tester('stringy!')), 'String');
+    });
+    it('of object Array is Array', function () {
+      assert.equal(util.getType(new Array([])), 'Array');
+    });
+    it('of object Date is Date', function () {
+      assert.equal(util.getType(new Date()), 'Date');
+    });
+    it('of object any other type is Object', function () {
+      assert.equal(util.getType({}), 'Object');
+    });
+    it('of number is Number', function () {
+      assert.equal(util.getType(1), 'Number');
+    });
+    it('of boolean is Boolean', function () {
+      assert.equal(util.getType(true), 'Boolean');
+    });
+    it('of string is String', function () {
+      assert.equal(util.getType('string'), 'String');
+    });
+    it('of undefined is undefined', function () {
+      assert.equal(util.getType(), 'undefined');
+    });
+  });
+
+  describe('easingFunctions', function () {
+    it('take a number and output a number', function () {
```

Comment:
What are bad inputs? Tests possible?

Reply:
These tests are to verify that all easing functions satisfy the required interface. Ie, accepts numeric input and provides numeric output.
Failures of any other type, or acceptance of any other type are irrelevant, imo.